frontend/src/pages/UserDetails.tsx

```diff
@@ -150,7 +150,26 @@ const UserDetailsPage: React.FC = () => {
               </div>
             </div>
             <div className="px-6 py-6">
-              {user.bio && <p className="text-lg text-gray-700 dark:text-gray-300">{user.bio}</p>}
+              {user.bio && (
+                <p className="text-lg text-gray-700 dark:text-gray-300">
+                  {user.bio.split(' ').map((word, index) => {
```

Comment:
If there's a dot after the link like in this example - the dot is added to the link, which makes it a `404`. 
![Screenshot 2025-03-01 at 5 58 41 PM](https://github.com/user-attachments/assets/8a9e2644-6003-4218-8667-d9be7872093d)
There could also be a `;!?` or any other thing like that. 
Could you fix this?